build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -216,8 +219,6 @@
       <Copy SourceFiles="@(MSBuildTargetsToCopy)"
             DestinationFiles="@(MSBuildTargetsToCopy->'$(SdkOutputDirectory)/%(RecursiveDir)%(Filename)%(Extension)')" />
 
-      <RemoveDir Directories="$(SdkOutputDirectory)/Extensions" />
```

Comment:
Have you seen and what is inside that folder besides your files? And how big that is? I think the right thing here is to still remove all the other files and just leave yours.